src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -584,7 +587,11 @@ void FullyConnected::createDescriptorInternal(const dnnl::memory::desc &inputDes
     dnnl::memory::data_type bdt = outdt;
 
     if (one_of(indt, dnnl::memory::data_type::bf16, dnnl::memory::data_type::f16)) {
+#if defined(OPENVINO_ARCH_X86_64)
         bdt = dnnl::memory::data_type::f32;
+#else
+        bdt = dnnl::memory::data_type::f16;
```

Comment:
please add comment: "oneDNN ARM InnerProduct primitive supports only identical in/out data types"